### PR TITLE
coreference resolution 

### PR DESCRIPTION
**Pull request by ansjin**
replacing the corefernces with their actual references in the text

### CODE REVIEW

**Review by Sandr0x00 — APPROVED** [review]
I have no write access ;)

File: algorithms/coreference_resolution/.swagger-codegen-ignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```
Comment (kordianbruck) [inline algorithms/coreference_resolution/.swagger-codegen-ignore]: These comments are really not needed!

**Comment by kordianbruck** [comment]
Why are you using shrinkwrap? https://github.com/MusicConnectionMachine/RelationshipsG3/blob/d586748e1027bd9925ae7a4f7b7a37aaab793b73/algorithms/coreference_resolution/npm-shrinkwrap.json

Everyone else is using: https://yarnpkg.com/lang/en/

**Comment by Sandr0x00** [comment]
Info for everybody: Is continued in https://github.com/MusicConnectionMachine/Relationships/pull/13

**Comment by sacdallago** [comment]
So Close @Sandr00 ?

**Comment by Sandr0x00** [comment]
Yes, but i can't

**Comment by sacdallago** [comment]
Sorry @Sandr00 :) It was really just a question, didn't know you didn't have write access

**Pull request closed without merging**